operator/api/v1alpha1/projectconfig_types.go

```diff
@@ -166,6 +170,15 @@ func (p *ProjectConfig) GetIngressClassName() string {
 	return p.Platform.IngressClassName
 }
 
+// GetCustomDomainIngressClassName returns the ingress class name for custom domain ingress.
+// if not set, return the common ingress class name
```

Comment:
这个默认值逻辑，是否应该在 CustomDomainIngressClassName 字段说明部分（公开文档）体现？

Reply:
在 CustomDomainIngressClassName 注释部分进行了说明。
```
// CustomDomainIngressClassName works the same as IngressClassName, but only for ingress with custom domains.
// Set customDomainIngressClassName if custom domain ingress needs to bound to a specific ingress controller,
// otherwise it will be bound to the ingress controller specified by ingressClassName config
```